modules/virt-setting-policy-attributes.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc
+
+[id="virt-setting-policy-attributes_{context}"]
+= Setting a policy attribute and CPU feature
+
+You can set a policy attribute and CPU feature for each virtual machine (VM) to ensure that it is scheduled on a node according to policy and feature. The CPU feature that you set is verified to ensure that it is supported by the host CPU or emulated by the hypervisor.
+
+.Prerequisites
+* You configured CPU node discovery.
+
+.Procedure
+
+. Edit the `domain` spec of your VM configuration file. The following example sets the CPU feature and the `require` policy for a virtual machine instance (VMI):
++
+[source,yaml]
+----
```

Comment:
Identation and content of the yaml are wrong, it should look like this:
```
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachineInstance
metadata:
  name: myvmi
spec:
  domain:
    cpu:
      features:
      - name: apic <1>
        policy: require <2>
```

Reply:
Fixed.